### PR TITLE
GW 7

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal-temp/modernized/07-chapter-7.adoc [hunk 1]
```diff
@@ -342,7 +342,13 @@ have room to lodge apart from the felons.
 
 A copy of the court order follows.
 
+[.embedded-content-document.legal]
+--
+
+[.signed-section-context-open]
```
Comment (maijapapaya) [inline journal-temp/modernized/07-chapter-7.adoc:349]: I wasn't sure about this like this.

File: journal-temp/modernized/07-chapter-7.adoc [hunk 2]
```diff
@@ -754,19 +764,30 @@ were often remembered; Who shall separate us from the love of Christ?
 Shall tribulation, or distress, or persecution, or famine, or nakedness, or peril,
 or sword?
 As it is written.
+
+[quote.scripture, , Rom. 8:35-39]
```
Comment (maijapapaya) [inline journal-temp/modernized/07-chapter-7.adoc:769]: I wasn't sure if I should do this either, but it's a longer quote.

File: journal-temp/modernized/07-chapter-7.adoc [hunk 3]
```diff
@@ -948,19 +973,18 @@ and the Lord my God preserved me both from the distemper and from banishment,
...
-The second epistle, printed in the year 1665, after the heat of the contagion was over,
-entitled, An epistle for the remnant of Friends and chosen of God,
-whom he has yet preserved to bear their testimony, in and about the city of London.
-To whom this is a testimony of the dear love and tender care which flows forth,
-and is extended toward them,
-and all tender hearts who are concerned in the like sufferings, temptations and trials.
+[.embedded-content-document.epistle]
+--
+
+[.blurb]
```
Comment (maijapapaya) [inline journal-temp/modernized/07-chapter-7.adoc:980]: So this part is a mess. haha I took some time to try to figure out what was happening here, and what I came up with is that it's somewhat of an introduction to the epistle maybe? Idk.

**Comment by Henderjay** [comment]
yeah, that super long letter heading thing was weird, especially because he signed the heading. I looked at it for a bit and couldn't think of anything better to do with it, so I just left it. 👍🏻 Merged

**Pull request merged**